log/DESIGN.md

```diff
@@ -0,0 +1,301 @@
+# Logs Bridge API
+
+## Abstract
+
+`go.opentelemetry.io/otel/log` provides
+[Logs Bridge API](https://opentelemetry.io/docs/specs/otel/logs/bridge-api/).
+
+The initial version of the design and the prototype
+was created in [#4725](https://github.com/open-telemetry/opentelemetry-go/pull/4725).
+
+## Background
+
+The key challenge is to create a well-performant API compliant with the specification.
+Performance is seen as one of the most important characteristics of logging libraries in Go.
+
+## Design
+
+This proposed design aims to:
+
+- be specification compliant,
+- be similar to Trace and Metrics API,
+- take advantage of both OpenTelemetry and `slog` experience to achieve acceptable performance.
+
+### Module structure
+
+The Go module consits of the following packages:
+
+- `go.opentelemetry.io/otel/log`
+- `go.opentelemetry.io/otel/log/embedded`
+- `go.opentelemetry.io/otel/log/noop`
```

Comment:
These should not be their own modules similar to tracing and metrics:

- https://pkg.go.dev/go.opentelemetry.io/otel/metric@v1.21.0/embedded
- https://pkg.go.dev/go.opentelemetry.io/otel/metric@v1.21.0/noop
- https://pkg.go.dev/go.opentelemetry.io/otel/trace@v1.21.0/noop
- https://pkg.go.dev/go.opentelemetry.io/otel/trace@v1.21.0/embedded

Reply:
Yes. One module with multiple packages.